core/3d/CCBundle3DData.h

```diff
@@ -43,6 +43,9 @@
 
 NS_CC_BEGIN
 
+using uint16_index_format  = std::bool_constant<true>;
+using uint32_index_format  = std::bool_constant<true>;
```

Comment:
both are true?

Reply:
Yes it's just the type that we care about

Reply:
Oh I see